src/AppBundle/Resources/config/doctrine/Card.orm.yml

```diff
@@ -44,7 +44,8 @@ AppBundle\Entity\Card:
             length: 50
             nullable: false
         cost:
-            type: smallint
+            type: string
+            length: 1
```

Comment:
While this change works on the current set of cards, it would not be long enough if a similar technique was applied to the STR field (which can also be X but can go into the double digits e.g. Mag the Mighty with STR 11). It seems somewhat unlikely they'll release a 10+ cost card when the current max is 8 cost, but it may be safer overall just to make this a 2 character string now. Your call.